scriptmodules/emulators/linapple.sh

```diff
@@ -37,31 +36,18 @@ function install_linapple() {
         'INSTALL'
         'LICENSE'
         'linapple'
-        'charset40.bmp'
-        'font.bmp'
-        'icon.bmp'
-        'splash.bmp'
         'Master.dsk'
         'README'
+        'README-linapple-pie'
+        'linapple.conf'
     )
     # install linapple.conf under another name as we will copy it
     cp -v "$md_build/linapple.conf" "$md_inst/linapple.conf.sample"
+    cp -vf "$md_build/Master.dsk" "$md_conf_root/apple2/Master.dsk"
 }
```

Comment:
yeah it is needed for dabones changes but I can move it to the configure function